server/storage/wal/repair_test.go

```diff
@@ -42,86 +45,59 @@ func TestRepairTruncate(t *testing.T) {
 }
 
 func testRepair(t *testing.T, ents [][]raftpb.Entry, corrupt corruptFunc, expectedEnts int) {
+	lg := zaptest.NewLogger(t)
 	p := t.TempDir()
 
 	// create WAL
-	w, err := Create(zaptest.NewLogger(t), p, nil)
+	w, err := Create(lg, p, nil)
 	defer func() {
-		if err = w.Close(); err != nil {
-			t.Fatal(err)
-		}
+		// The Close might fail.
+		_ = w.Close()
 	}()
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	for _, es := range ents {
-		if err = w.Save(raftpb.HardState{}, es); err != nil {
-			t.Fatal(err)
-		}
+		assert.NoError(t, w.Save(raftpb.HardState{}, es))
```

Comment:
Did you intentionally use `assert.NoError` instead of `require.NoError`?

Reply:
Changed to require... not crucial for the rest of the test... but further failures might be misleading.